pipeline/processor/postprocessor.py

```diff
@@ -169,13 +169,15 @@ def merge_text_blocks(self, text_blocks: List[dict]) -> dict:
 
             # Append style metadata that will later be added to custom attributes for text block.
             if block_styling:
-                last_style_ix = len(merged_block_text[0].strip()) - 1
-                style_spans.append(
-                    {
+                # The last style index here is the index of the block with sentences joined.
+                last_style_ix = len(''.join((text for text in merged_block_text)).strip()) - 1
+                style_span = {
```

Comment:
This is a simple bug fix I noticed where I failed to account for the the length of the whole joined string in a list of strings, which was messing up some of the Spacy style indexing for style blocks, so needed an upstream fix.